emsdk.py

```diff
@@ -340,15 +338,13 @@ def win_get_environment_variable(key, system=True):
       py_path = to_native_path(py.expand_vars(py.activated_path))
       os.environ['PATH'] = os.environ['PATH'] + ';' + py_path
```

Comment:
Out of interest do you have any insight as to why we over ride `$PATH` during this function?

Reply:
Good catch. I think we don't need this anymore! This is adding the python that ships with emsdk to PATH to perform win32api operations. Considering that it is removed, we can get rid of these.

Reply:
https://github.com/emscripten-core/emsdk/pull/635/commits/990f370945bc829c005bb7c5619ec1e7780db37f